### PR TITLE
Extended README to contain more infromation on how to build the homepage

### PR DESCRIPTION
**Pull request by ajaust**
Add information about software dependencies, repository structure and how to build the homepage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+
+The tools used are all based on Python 3 so the easiest way to install the theme and all dependencies is to use [pip](https://pypi.org/project/pip/). Running
+
+```bash
+pip install mkdocs-material
+```
+
+will install the MkDocs, theme, and the plugins.
+
+If you use an operating system with  `pip` point
```
Comment (uekerman) [inline README.md:33]: Something went wrong with this sentence :grin:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
 # Homepage for lecture "Simulation Software Engineering"
 
-More content is coming soon!
\ No newline at end of file
+![publish-homepage](https://github.com/Simulation-Software-Engineering/homepage/actions/workflows/ci.yml/badge.svg)
+
+This repository contains the content of for the homepage of the "Simulation Software Engineering" lecture and University of Stuttgart (Germany). The homepage is built on [mkdocs](https://www.mkdocs.org), the ["Material for MkDocs" theme](https://squidfunk.github.io/mkdocs-material/), and corresponding plugins.
```
Comment (uekerman) [inline README.md:5]:
```suggestion
This repository contains the content of for the homepage of the "Simulation Software Engineering" lecture at University of Stuttgart (Germany). The homepage is built on [mkdocs](https://www.mkdocs.org), the ["Material for MkDocs" theme](https://squidfunk.github.io/mkdocs-material/), and corresponding plugins.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+## Structure of the repository
+
+- `mkdocs.yml`: MkDocs configuration file
+- `docs/`: The directory contains the Markdown files for the different pages of the homepage.
+- `.github/workflows/ci.yml`: Definition file of GitHub action which builds the homepage on every successfull push.
+- `README.md`: The Readme file you are currently reading
+
+## Build homepage
+
+**Dependencies**
```
Comment (uekerman) [inline README.md:17]:
```suggestion
###Dependencies
```

Some markdown checkers will complain about such fake headlines IIRC.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+pip install mkdocs-material
+```
+
+will install the MkDocs, theme, and the plugins.
+
+If you use an operating system with  `pip` point
+
+Alternative ways of building the homepage using MkDocs and Material for MkDocs are described in the [documentation of the theme](https://squidfunk.github.io/mkdocs-material/getting-started/).
+
+**Building the homepage locally**
```
Comment (uekerman) [inline README.md:37]:
```suggestion
###Building the homepage locally
```

**Review by uekerman — APPROVED** [review]
Works like a charm. Thanks!
And sorry for the slow reaction.

File: README.md (hunk 1 above)
Comment (ajaust) [reply README.md]: I removed the sentence as I did not see what it should be good for.

File: README.md (hunk 2 above)
Comment (ajaust) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (ajaust) [reply README.md]: Done.

**Pull request merged**